samples/csharp/uwp/virtualassistant-uwp/README.md

```diff
@@ -1,13 +1,11 @@
-# Quickstart: Using virtual assistant (Preview) in C# Universal Windows Platform
+# Quickstart: Using Voice Assistant in C# Universal Windows Platform
```

Comment:
I'd recommend the plural of "Voice Assistants" for the switch

Reply:
Sure!